docs/aws-china.md

```diff
@@ -0,0 +1,261 @@
+# How to use kops in AWS China Region
+
+## Getting Started
+
+Kops used to only support Google Cloud DNS and Amazon Route53 to provision a kubernetes cluster. But since 1.6.2 `gossip` has been added which make it possible to provision a cluster without one of those DNS providers. Thanks to `gossip`, it's officially supported to provision a fully-functional kubernetes in AWS China Region [which doesn't have Route53 so far][1] since [1.7][2].
```

Comment:
Can we provide the name of the region? Also how much work would it be to get this doc in mandarin?

Reply:
> Also how much work would it be to get this doc in mandarin?

Maybe this weekend or next...